packages/x-charts/src/BarChart/BarLabel/types.ts

```diff
@@ -0,0 +1,46 @@
+import { SeriesId } from '../../models/seriesType/common';
+import type { BarLabelClasses } from './barLabelClasses';
+
+export interface BarLabelOwnerState {
+  seriesId: SeriesId;
+  dataIndex: number;
+  color: string;
+  isFaded: boolean;
+  isHighlighted: boolean;
+  classes?: Partial<BarLabelClasses>;
+}
+
+export type BarItem = {
```

Comment:
Just a comment: We could start having those `XxxItem` types which will be a good complement with the `XxxItemIdentifier` in the `models/seriesType/`

Reply:
I don't get what you mean 🤔 

I didn't see that type as well, but I'm not sure the `type:'bar'` is necessary here

Reply:
> I didn't see that type as well

That's the point I wanted to point this type such that you know they exist for all charts.

> but I'm not sure the type:'bar' is necessary here

I agree it was just to share information